src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -168,38 +168,45 @@ public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub
                         _logger.LogDebug("Received hub invocation: {invocation}", invocationDescriptor);
                     }
 
-                    InvocationResultDescriptor result;
-                    Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>> callback;
-                    if (_callbacks.TryGetValue(invocationDescriptor.Method, out callback))
-                    {
-                        result = await callback(connection, invocationDescriptor);
-                    }
-                    else
-                    {
-                        // If there's no method then return a failed response for this request
-                        result = new InvocationResultDescriptor
-                        {
-                            Id = invocationDescriptor.Id,
-                            Error = $"Unknown hub method '{invocationDescriptor.Method}'"
-                        };
+                    // Don't wait on the result of execution, continue processing other
+                    // incoming messages on this connection.
+                    var ignore = Execute(connection, invocationAdapter, invocationDescriptor);
```

Comment:
unexpected exceptions should end the connection indeed. but because we are not awaiting the task returned by `Execute` it will be faulted and the exception will not propagate right?